source/routes/read/daily.js

```diff
@@ -0,0 +1,52 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7422202)
```suggestion
const db = new PouchDB("db");
```